### ISSUE TITLE
Issue with the Code of Conduct in Readme.md redirecting to a `422: Unprocessable Entity`

### ISSUE THREAD

**Issue by gSayak** [rust-lang/cargo#13407]
### Problem

Right now the **code-of-conduct** link in the `Readme.md` in the `master` branch redirects to:
 `https://www.rust-lang.org/conduct.html` 

which results in an:
``` 
**422: Unprocessable Entity
The request was well-formed but was unable to be followed due to semantic errors.**
```
It should have redirected to: `https://www.rust-lang.org/policies/code-of-conduct`

Fixed this issue in the Readme.md by making a new branch and will be linking the PR to this issue

### Steps

1. Go to `https://github.com/rust-lang/cargo`
2. Check the readme.md and click on Code of Conduct
3. Click on the hyperlink of `can be found online`
4. This would redirect to a **422: Unprocessable Entity**

### Possible Solution(s)

Can fix the issue by changing the link to [Code-of-Conduct](https://www.rust-lang.org/policies/code-of-conduct)

**Comment by gSayak** [comment]
Solved the issue and sent the PR #13408

**Comment by weihanglo** [comment]
This has been fixed in <https://github.com/rust-lang/www.rust-lang.org/pull/1924>.

Redirections in staging website work now: <https://www-staging.rust-lang.org/conduct.html>. When the next deployment of the production website is done, this should be resolved. Thank you for the report!
